api/proto/info/info.proto

```diff
@@ -183,6 +183,13 @@ enum HwSecurityModuleStatus {
     ENABLED  = 3;  //HSM is found and being actively used
 }
 
+enum DeviceEncryptionStatus {
+    ESP_TYPE_NONE = 0;        // Data is not encrypted
+    ESP_TYPE_RESTDATA = 1;    // Data at rest encryption enabled
+    ESP_TYPE_TRANSITDATA = 2; // Data at transit encryption enabled
+    ESP_TYPE_ALLDATA = 3;     // All sesitive data is encrypted
+}
```

Comment:
This is misleading since everything is already encrypted in transit using TLS1.2.